storage/imxrt-flash/flashsrv.c

```diff
@@ -534,6 +534,16 @@ static void flashsrv_rawThread(void *arg)
 				msg.o.err = flashsrv_bufferedSync(memory->fOid.id);
 				break;
 
+			case mtGetAttr:
+				if (msg.i.attr.type == atSize) {
+					msg.o.attr.val = memory->parts[msg.oid.id].pHeader->size;
```

Comment:
Oh, I thing `msg.oid.id` should be checked before `switch` if it's valid (`memory->parts[]` potential out of bounds access) when a message with an invalid `oid.id` would be sent.

Reply:
Thanks, done